### PR TITLE
fix: Fix trigger maintenance and remove buttons on triggers list

### PR DESCRIPTION
**Pull request by borovskyav**
Fix https://github.com/moira-alert/moira/issues/358

### CODE REVIEW

File: src/pages/trigger-list/trigger-list.jsx [hunk 1]
```diff
@@ -193,6 +198,31 @@ class TriggerListPage extends React.Component<Props, State> {
             })}`
         );
     };
+
+    setMetricMaintenance = async (triggerId: string, maintenance: Maintenance, metric: string) => {
```
Comment (beevee) [inline src/pages/trigger-list/trigger-list.jsx]: Can you merge this code with similar code that works on trigger page?

Comment (borovskyav) [reply src/pages/trigger-list/trigger-list.jsx]: I have check all duplications, found unused code. Fixed it.

File: src/pages/trigger-list/trigger-list.desktop.jsx [hunk 2]
```diff
@@ -55,8 +57,12 @@ class TriggerListDesktop extends React.Component {
                         <TriggerList
                             searchMode={searchText !== ""}
                             items={triggers}
-                            onChange={() => {}}
-                            onRemove={() => {}}
+                            onChange={(triggerId, metric, maintenance) => {
+                                onSetMetricMaintenance(triggerId, metric, maintenance);
+                            }}
```
Comment (sashasushko) [inline src/pages/trigger-list/trigger-list.desktop.jsx]: `onChange={onSetMetricMaintenance}`

Comment (sashasushko) [reply src/pages/trigger-list/trigger-list.desktop.jsx]: Обёртка не нужна

File: src/pages/trigger-list/trigger-list.desktop.jsx [hunk 3]
```diff
@@ -55,8 +57,12 @@ class TriggerListDesktop extends React.Component {
...
+                            onRemove={(triggerId, metric) => {
+                                onRemoveMetric(triggerId, metric);
+                            }}
```
Comment (sashasushko) [inline src/pages/trigger-list/trigger-list.desktop.jsx]: Аналогично — `onRemove={onRemoveMetric}`

File: src/pages/trigger-list/trigger-list.desktop.jsx (hunk 2 above)
Comment (borovskyav) [reply src/pages/trigger-list/trigger-list.desktop.jsx]: выпилил

File: src/pages/trigger-list/trigger-list.desktop.jsx (hunk 3 above)
Comment (borovskyav) [reply src/pages/trigger-list/trigger-list.desktop.jsx]: готово

**Pull request merged**